clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -337,7 +337,7 @@ def warn_atomic_op_oversized : Warning<
 InGroup<AtomicAlignment>;
 
 def warn_sync_op_misaligned : Warning<
-  "__sync builtin operation MUST have natural alignment (consider using __atomic).">,
+  "__sync builtin operation MUST have natural alignment (consider using __atomic)">,
```

Comment:
So, it is not allowed to capitalize a message, but a whole word written (`MUST` in this example) with uppercase is allowed? Isn't it too much?